paimon-core/src/main/java/org/apache/paimon/stats/FieldStatsConverters.java

```diff
@@ -37,20 +38,27 @@ public class FieldStatsConverters {
     private final ConcurrentMap<Long, FieldStatsArraySerializer> serializers;
     private final AtomicReference<List<DataField>> tableFields;
 
-    public FieldStatsConverters(Function<Long, List<DataField>> schemaFields, long tableSchemaId) {
+    Set<Integer> skipFieldIndex;
+
+    public FieldStatsConverters(
```

Comment:
Maybe just modify this to:
```
public FieldStatsConverters(
            Function<Long, List<DataField>> schemaFields,
            TableSchema currentSchema) 
```

Reply:
OK,How about adding an additional switch arg to control whether to skip the statistics information? Previously, passing the default value from outside the method was intended to display the statistics information for fields even with default values in FilesTable.